### PR TITLE
Adding ARGs in CmdConstant

### PR DESCRIPTION
**Pull request by bjlee2000**

### CODE REVIEW

File: pytwis/pytwis_clt.py [hunk 1]
```diff
@@ -223,20 +230,20 @@ def pytwis_command_parser(raw_command):
         arg_dict = parse.parse('{username} {password}', args)
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: Could you please replace '{username} {password}' by the ARG constants?

```python
arg_dict = parse.parse('{{{arg1}}} {{{arg2}}}'.format(arg1=ARG_USERNAME, arg2=ARG_PASSWORD), args)
```

File: pytwis/pytwis_clt.py [hunk 2]
```diff
@@ -223,20 +230,20 @@ def pytwis_command_parser(raw_command):
...
             raise ValueError("password can't contain spaces")
 
         print('{}: username = {}, password = {}'.format(CmdConstant.CMD_REGISTER, \
-                                                        arg_dict['username'], arg_dict['password']))
-    elif command_with_args[0] == 'login':
+                                                        arg_dict['username'], arg_dict[CmdConstant.ARG_PASSWORD]))
+    elif command_with_args[0] == CmdConstant.CMD_LOGIN:
         # login must have two arguments: username and password.
         args = splited_raw_command[1]
         arg_dict = parse.parse('{username} {password}', args)
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: Same as above.

File: pytwis/pytwis_clt.py [hunk 3]
```diff
@@ -246,13 +253,13 @@ def pytwis_command_parser(raw_command):
         arg_dict = parse.parse('{old_password} {new_password} {confirmed_new_password}', args)
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: Same as above.

File: pytwis/pytwis_clt.py [hunk 4]
```diff
@@ -246,13 +253,13 @@ def pytwis_command_parser(raw_command):
...
         if arg_dict is None:
             raise ValueError('{} has incorrect arguments'.format(CmdConstant.CMD_CHANGE_PASSWORD))
-        elif arg_dict['new_password'] != arg_dict['confirmed_new_password']:
+        elif arg_dict[CmdConstant.ARG_NEW_PASSWORD] != arg_dict['confirmed_new_password']:
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: Define an ARG constant for 'confirmed_new_password'?

File: pytwis/pytwis_clt.py [hunk 5]
```diff
@@ -261,10 +268,10 @@ def pytwis_command_parser(raw_command):
         arg_dict = {'tweet': splited_raw_command[1]}
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: Define an ARG constant for 'tweet'?

File: pytwis/pytwis_clt.py [hunk 6]
```diff
@@ -223,20 +230,20 @@ def pytwis_command_parser(raw_command):
...
         if arg_dict is None:
             raise ValueError('{} has incorrect arguments'.format(CmdConstant.CMD_REGISTER))
-        elif ' ' in arg_dict['password']:
+        elif ' ' in arg_dict[CmdConstant.ARG_PASSWORD]:
             raise ValueError("password can't contain spaces")
 
         print('{}: username = {}, password = {}'.format(CmdConstant.CMD_REGISTER, \
-                                                        arg_dict['username'], arg_dict['password']))
-    elif command_with_args[0] == 'login':
+                                                        arg_dict['username'], arg_dict[CmdConstant.ARG_PASSWORD]))
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: arg_dict[CmdConstant.ARG_USERNAME]?

Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: Fixed.

File: pytwis/pytwis_clt.py (hunk 4 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: Fixed

File: pytwis/pytwis_clt.py (hunk 1 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: Fixed

File: pytwis/pytwis_clt.py (hunk 2 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: Fixed.

File: pytwis/pytwis_clt.py (hunk 3 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: Fixed

**Comment by bjlee2000** [comment]
The review comments are addressed.

**Pull request merged**